tinyiiod.c

```diff
@@ -96,7 +93,7 @@ ssize_t tinyiiod_read_line(struct tinyiiod *iiod, char *buf, size_t len)
 
 	buf[i] = '\0';
 
-	return (ssize_t) i;
+	return i;
```

Comment:
hmm.. is this intended? This function returns `uint32_t `instead of `int32_t` as it would be expected from the function type.

Reply:
I will change 
uint32_t i; 
to 
int32_t i;